meta/store.go

```diff
@@ -868,6 +868,10 @@ func (s *Store) CreateDatabase(name string) (*DatabaseInfo, error) {
 			return nil, fmt.Errorf("read: %s", err)
 		}
 
+		if nodeN > 3 {
```

Comment:
No.  This is the auto-created retention policy.  It just caps the replication factor at 3 instead of using the number of nodes.  You can manually create replication factors greater than 3 now.